test/browser/clubs-page.test.jsx

```diff
@@ -9,28 +9,9 @@ var StubTeachAPI = require('./stub-teach-api');
 var StubRouter = require('./stub-router');
```

Comment:
moved relevant tests to the corresponding files, i.e., `modal-clubs-remove.test.jsx` and 
`modal-clubs.test.js`